### PR TITLE
#164069326 Reporting an article 

### PR DESCRIPTION
**Pull request by akram256**
**What does this PR do?**
* Enables users to report an article

**Description of the tasks to be completed?**
* Writing tests  for  report an article
* Implement send_mail after an article has been reported

**How should this be manually tested?**
* Create an article `POST api/article`
* Select an article slug that you want to comment on
* Report an article  `POST api/article/slug/report/`

Screenshot 2019-03-18 at 10 57 36

 **When an article is reported, an email is sent to the Admin.**
Screenshot 2019-03-18 at 10 59 08

**What are the relevant pivot tracker stores?**
 #164069326

### CODE REVIEW

**Review by NLSanyu — APPROVED** [review]

File: authors/apps/articles/views.py [hunk 1]
```diff
@@ -355,3 +361,59 @@ class ListTagsView(ListAPIView):
     permission_classes = (IsAuthenticated,)
     renderer_classes = (TagJSONRenderer,)
     queryset = Tag.objects.all()
+
+
+class ReportView(CreateAPIView):
```
Comment (walimike) [inline authors/apps/articles/views.py]: Rename this class to CreateReportView

**Review by walimike — APPROVED** [review]

File: authors/apps/articles/views.py (hunk 1 above)
Comment (akram256) [reply authors/apps/articles/views.py]: Okay 
Thanks @walimike

**Comment by nyabongoedgar** [comment]
@akram256 , on what does this PR do ?, you say it reports an article that violates an article.
That does not sound right, it should be something like the terms and conditions or regulations of writing an article in regards to its content.

**Review by Stanley-Okwii — APPROVED** [review]
👏🏾👏🏾👏🏾👏🏾👏🏾👏🏾👏🏾👏🏾👏🏾
👏🏾👏🏾👏🏾👏🏾👏🏾👏🏾👏🏾👏🏾👏🏾

**Review by PatrickCmd — APPROVED** [review]

**Pull request merged**